### ISSUE TITLE
SQLite - Failing to import CSV with column named Plan

### ISSUE THREAD

**Issue by Zulu-Inuoe** [dbeaver/dbeaver#13888]
#### System information: 
- Windows 10 x64
- DBeaver 21.1.5

#### Connection specification:
- SQLite 3.36.0.3

#### Describe the problem you're observing:
When importing data from a CSV file, fails to import/preview data when there is a column named `Plan`.

The error reported is:

```
Can't load entity attributes

Reason:
Can't find target attribute [Plan]
```

The CREATE TABLE expression properly escapes `Plan` as `"Plan"`, so seeing it brackets here is odd. SQLite does not support bracket-escaping like SQL Server etc does.

#### Steps to reproduce, if exist:

Attempt to import a CSV file with a Plan column, eg:

create a file with these contents:
```csv
id,Plan
1029,Something Here
```

then right-click on a SQLite database connection and select "Import", import from CSV.
Then on the column mapping window, select "Preview Data"

#### Include any warning/errors/backtraces from the logs
```
Can't load entity attributes

Reason:
Can't find target attribute [Plan]
```

**Comment by LonwoLonwo** [comment]
Hello @Zulu-Inuoe 
Thanks for the bug report.

Problem was fixed here: #13402, #13542

You can download [DBeaver's last version](https://dbeaver.io/download/) and test it.

**Comment by Zulu-Inuoe** [comment]
Oh sorry about the duplicate I didn't realize 21.2 had been released.

Thanks!
